app/javascript/app/providers/ndcs-sdgs-meta-provider/ndcs-sdgs-meta-provider-actions.js

```diff
@@ -4,6 +4,7 @@ import isEmpty from 'lodash/isEmpty';
 
 const getNdcsSdgsMetaInit = createAction('getNdcsSdgsMetaInit');
 const getNdcsSdgsMetaReady = createAction('getNdcsSdgsMetaReady');
+const getNdcsSdgsMetaFail = createAction('getTimelineFail');
```

Comment:
Fail is the same as Ready, but with {} instead of data as the payload. Can we not reuse Ready in each of these situations?